docs/learn/wallets/firefly-wallet.md

```diff
@@ -1,83 +1,87 @@
 ---
 title: Firefly Wallet
-description: The IOTA Firefly Wallet sets a new standard for Software Wallets in DLT. We describe why it is so secure and how amazing it is to use it.
+description: The IOTA Firefly Wallet sets a new standard for Software Wallets in
+  DLT. Learn everything here.
 ---
 
-
 # The Firefly Wallet
 
+## Table of contents
+
+- General Overview and Introduction 
```

Comment:
Please don't use absolute links in the ToC. It will open a new tab in most browsers 😅 
I will commit a new ToC. 
Tell me. if you like it :)

Reply:
<img width="634" alt="ToC" src="https://user-images.githubusercontent.com/61637085/134067574-1cb9a566-6c79-4b97-ad9d-365534998224.png">

Reply:
me like